docs/imviz/plugins.rst

```diff
@@ -269,6 +269,22 @@ documented in :ref:`astropy:astropy-table`.
 
 .. _imviz-export-plot:
```

Comment:
This line defines an alias for subsequent section; I think you should move your Catalog Search docs section above this line

Reply:
Thanks, just did it! Does my section need an alias as well?

Reply:
It allows it to be referenced elsewhere in the `docs` directory, so can't hurt.